scripts/run_mypy_checks.py

```diff
@@ -39,7 +39,8 @@
     # ignored while type checking, because these files are only
     # used as resources for the tests.
     'core/tests/build_sources/',
-    'core/tests/data/'
+    'core/tests/data/',
+    'src_graphql/'
```

Comment:
The comment mentioned above is ok for files ( or directories) like `test/.` but now you have added `src_graphql` directory which I don't think is aligned with the above comment. Please add a comment on why this directory needs to be excluded from the backend type checks.